src/only_lists.rb

```diff
@@ -24,7 +24,8 @@ def list_rentals(db_persons)
     print "\nID of the person: "
     person_id = gets.chomp.to_i
     person_fetch = db_persons.select { |person| person.id == person_id }
-    if person_fetch[0].id
+    # if person_fetch[0] && person_fetch[0].id
```

Comment:
- [ ] Kindly remove all commented out, un used. unwanted pieces of code from your code base. It makes it look messy and cluttered.

Reply:
@Nemwel-Boniface Done !